pages/desktop/consumer_pages/base.py

```diff
@@ -67,13 +70,22 @@ def set_region(self, region):
 
         self.wait_notification_box_not_visible()
 
-    def login(self, user=None):
-        from pages.fxa import FirefoxAccounts
-        fxa = FirefoxAccounts(self.testsetup)
-        fxa.login_user(user)
+    def login(self, email, password):
```

Comment:
Thanks for making this change, @AndreiH. I think that the `login` method could accept a user object, rather than the `email` and `password. You have set up  `create_new_user` and `get_default_user` to both return a `MockUser` object, so it would be fine for `login` to expect this object. That would mean that the tests wouldn't have to extract the `email` and `password` from the user.